htdocs/core/actions_massactions.inc.php

```diff
@@ -1133,6 +1134,30 @@
 	//var_dump($listofobjectthirdparties);exit;
 }
 
+// close cashpoint from mass action
```

Comment:
if you need to add a mass action that is specific to a page/object, you must add it into the file compta/cashcontrol/cashcontrol_list.php
You can add this test just before the include of the generic mass action

Reply:
Modifications done